app/models/post.rb

```diff
@@ -35,4 +35,12 @@ def normalize_friendly_id(string)
   def should_generate_new_friendly_id?
     slug.blank? || title_changed?
   end
+
+  def related_posts(limit = 3)
```

Comment:
Currently just returns the three most recent related posts, do we want some randomness in here?

Reply:
yeah, I think a order by rand() would be nice.